backend/build.go

```diff
@@ -58,12 +58,17 @@ func Build(
 		"grafana",
 		"grafana-server",
 		"grafana-cli",
+		"grafana-example-apiserver",
 	}
 
 	for _, v := range cmd {
-		cmd := GoBuildCommand(path.Join(out, v), ldflags, opts.Tags, path.Join("pkg", "cmd", v))
+		// Some CLI packages such as grafana-example-apiserver don't exist in earlier Grafana Versions <10.3
+		// Below check skips building them as needed
+		pkgPath := path.Join("pkg", "cmd", v)
+
+		cmd := GoBuildCommand(path.Join(out, v), ldflags, opts.Tags, pkgPath)
 		builder = builder.
-			WithExec([]string{"/bin/sh", "-c", strings.Join(cmd, " ")})
+			WithExec([]string{"/bin/sh", "-c", fmt.Sprintf(`if [ -e %s ]; then %s; fi`, pkgPath, strings.Join(cmd, " "))})
```

Comment:
```suggestion
			WithExec([]string{"/bin/sh", "-c", fmt.Sprintf(`if [ -d %s ]; then %s; fi`, pkgPath, strings.Join(cmd, " "))})
```

my bad, use -d here instead of -e just to make sure it's a directory. With that change we can merge this!